packages/api/src/router/workspace.ts

```diff
@@ -41,16 +45,18 @@ export const workspaceRouter = createTRPCRouter({
         with: {
           pages: true,
         },
-        where: eq(workspace.id, opts.input.workspaceId),
+        where: eq(workspace.id, currentWorkspace.id),
       });
     }),
 
   createWorkspace: protectedProcedure
     .input(z.object({ name: z.string() }))
     .mutation(async (opts) => {
+      const slug = generateSlug(2);
```

Comment:
We might want to add a check later, if the generated `slug` already exists. Can be done later I think - would doubt if we get twice the same slug for a while